### PR TITLE
Drag animation

### PR DESCRIPTION
**Pull request by GabriellAnnunziata**

### CODE REVIEW

File: RayD/RayD/UI/MainView/Subviews/AnimationView.swift [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                     .onEnded {value in
+                         let height = value.translation.height
+                         if height > 60 {
+                             isStarted = false
+                             startedOffset = .zero
+                         }
+                     }
+                     )
+     }
+    var body: some View {
```
Comment (palant-dev) [inline RayD/RayD/UI/MainView/Subviews/AnimationView.swift:54]: Usually body is on the top as most important property, dunno if it's a formatting rule

**Review by palant-dev — APPROVED** [review]
Great job! I like the way you implemented the matched geometry.

File: RayD/RayD/UI/MainView/Subviews/AnimationView.swift [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+//
+//  Created by Gabriella Annunziata on 01/06/23.
+//
+
+import SwiftUI
+
+struct AnimationView: View {
+    @Namespace var namespace
+    @State var pausedSphereDimension: CGFloat = 138
+    @State var playedSphereDimension: CGFloat = 272
```
Comment (yukim17) [inline RayD/RayD/UI/MainView/Subviews/AnimationView.swift:13]: I suppose we should calculate width of the sphere based on screen width minus paddings. Try to use `UIScreen.main.bounds.width` to get screen width

Comment (palant-dev) [reply RayD/RayD/UI/MainView/Subviews/AnimationView.swift:13]: This can be a good idea to avoid difficulties when calling the struct in content view

**Review by slavny11 — APPROVED** [review]

**Review by yukim17 — APPROVED** [review]

**Pull request merged**